integrationExamples/gpt/pbjs_example_gpt.html

```diff
@@ -240,6 +240,13 @@
                             publisher_id: 5000563,  // REQUIRED int or str publisher ID. To get one, register at https://control.adequant.com
                             bidfloor: 0.01,         // OPTIONAL float bid floor in $ CPM
                         }
+                    },
+                    {
+                        bidder: 'wideorbit',
+                        params: {
+                            pbId: 123, // Publisher Id,
+                            pId: 123456 // Placement Id

```

Comment:
Have marked them as required. The usage of the parameters will be clear from the documentation. The user will also download those definitions from our UI or API, they will probably never create them manually.